### ISSUE TITLE
profiled track not appearing from cross-study queries

### ISSUE THREAD

**Issue by jjgao** [cBioPortal/cbioportal#4310]
could you make sure we show the additional tracks that indicate samples that are not covered by certain profiles. It seems not working for multi-study queries. e.g. 

http://www.cbioportal.org/beta/index.do?session_id=5b0ef717498eb8b3d5665c70

![image](https://user-images.githubusercontent.com/840895/40742064-4ff877fe-641b-11e8-9fef-0e781a455ac4.png)

**Comment by adamabeshouse** [comment]
@jjgao the tracks only show if there are samples that are not profiled

**Comment by adamabeshouse** [comment]
eg http://www.cbioportal.org/beta/index.do?cancer_study_id=all&Z_SCORE_THRESHOLD=2&RPPA_SCORE_THRESHOLD=2&data_priority=0&case_set_id=0&gene_list=EGFR%2520HCL1&geneset_list=%20&tab_index=tab_visualize&Action=Submit&cancer_study_list=acc_tcga%2Cchol_nccs_2013%2Cmsk_impact_2017&show_samples=false&clinicallist=CANCER_STUDY%2CPROFILED_IN_COPY_NUMBER_ALTERATION%2CPROFILED_IN_MUTATION_EXTENDED

**Comment by jjgao** [comment]
@adamabeshouse the samples from `Cholangiocarcinoma (National Cancer Centre of Singapore, Nat Genet 2013)` have no CNA data...

**Comment by adamabeshouse** [comment]
@jjgao for each sample we're only considering profiles from its study, because otherwise we'd be showing "Sample X from coadread_tcga_pub is not sequenced in impact mutations". I see that in this case though it makes sense to show if samples dont have CNA data...

**Comment by jjgao** [comment]
@adamabeshouse agreed that "for each sample we're only considering profiles from its study". In this case, we should have a track to show samples without CNA data...

When querying both mutation and CNA
![image](https://user-images.githubusercontent.com/840895/40747055-3028d2f2-642a-11e8-8a46-4fccda888174.png)

When query only CNA
![image](https://user-images.githubusercontent.com/840895/40747128-598d9c86-642a-11e8-932a-feb9c7b114ea.png)

**Comment by adamabeshouse** [comment]
okay I fixed the issue, but now we have a bit of a confusing situation: 

![image](https://user-images.githubusercontent.com/636232/40800468-82e0b3e8-64de-11e8-9262-4210ac8bb7e9.png)

![image](https://user-images.githubusercontent.com/636232/40800498-8f5333c6-64de-11e8-9093-a8084e5fa204.png)

the tooltip says "profiled in all selected molecular profiles", because it is profiled in all selected molecular profiles for its study, but the track above says its not profiled for copy number alterations

**Comment by adamabeshouse** [comment]
after discussing w @jjgao we decided to say "Not profiled in: CNA" in that case
